cloud/amazon/ec2.py

```diff
@@ -499,6 +500,15 @@
     state: running
 
 #
+# Restart instances specified by tag
+#
+- local_action:
+    module: ec2
```

Comment:
region is not commonly used in other examples like terminate and start, I think a follow up PR should be done including those everywhere required.